makefile

```diff
@@ -163,7 +163,7 @@ endif
 
 pretty-lines:
 	@git diff -U0 --no-color $(DIFFBASE) | python3 util/clang-format-diff.py -binary $(CLANG_FORMAT_BINARY) -i -p1
-	@git diff -U0 --no-color $(DIFFBASE) | python3 util/yapf-diff.py -style .style.yapf -i -p1
+	@git diff -U0 --no-color $(DIFFBASE) | black .
```

Comment:
Modified Makefile's pretty-lines command to run black on all files (excluding one's defined in config file) instead of yapf